src/app/admin/AdminPage.tsx

```diff
@@ -35,6 +45,18 @@ const AdminPage: React.FC<Props> = (props) => {
             panelContent: <UsersTable userData={props.userData} />,
         },
         { panelTitle: "CREATE USER", panelIcon: faUserPlus, panelContent: <CreateUserForm /> },
+        {
+            panelTitle: "COLLECTION CENTRES TABLE",
```

Comment:
can you change this this to be 'normal' case? Daniel is making a quick change to do the other panels - if you want upper case titles, set it in CSS rather than in the text!

Reply:
Sure